### PR TITLE
Added in new state for special project visits to the gauge (PS-381). …

### PR DESCRIPTION
**Pull request by neolinkpsu**
… Also changed the labeling to be more inline with what we're reporting out which is visits to the store

This is code to add in the new status Visited, but Incomplete. Part 2 of 2 (1 of 2 in ThirdChannel)

### CODE REVIEW

File: js/apps/thirdchannel/views/dashboards/special_projects/item.js [hunk 1]
```diff
@@ -20,28 +20,31 @@ define(function(require) {
...
-                }
-            } else {
-                data.push(['Completion', percentComplete]);
+            percentComplete += percentIncomplete;
+
+            data.push(['Target', target]);
+            data.push(['Completed', percentComplete]);
+            data.push(['Visited, but incompleted', percentIncomplete]);
+
+            if(percentComplete <= target) {
```
Comment (spullen) [inline js/apps/thirdchannel/views/dashboards/special_projects/item.js]: I think the reason why I had a guard around target was that some jobs don't specify it (maybe for the legacy case), might want to double check that.

Comment (spullen) [inline js/apps/thirdchannel/views/dashboards/special_projects/item.js]: I think the reason why I had a guard around target was that some jobs don't specify it (maybe for the legacy case), might want to double check that.

**Comment by spullen** [comment]
Just one thing to look into, other than that looks fine to me.

File: js/apps/thirdchannel/views/dashboards/special_projects/item.js (hunk 1 above)
Comment (neolinkpsu) [inline js/apps/thirdchannel/views/dashboards/special_projects/item.js]: /lib/dashboard_service.rb: line: 141.  We seem to always return a target value.  If it's null it's handled correctly on the front end

Comment (spullen) [inline js/apps/thirdchannel/views/dashboards/special_projects/item.js]: ok

**Pull request merged**